### DISCUSSION TITLE
Any estimates on v1 release date?

### DISCUSSION THREAD

**Discussion by giiyms** [fzyzcjy/flutter_smooth#184] [General]
Hello,

For some reason the discord is not working for me.
Just curious, how is the progress going? 
I see a lot of PRs are merged or closed. 

Any estimates of first pub.dev release?

**Comment by fzyzcjy** [comment]
Hi, yes many PRs have been merged or closed. I have provided demos that Google Flutter engineers want to see, discussed with them, etc. However, it is not a trivial action that they can take since this involves changing some very low-level things in Flutter, so they seem to be very careful before merging - which I can totally understand. Thus, I cannot provide any estimates :/

**Reply by AndryHTC** [reply]
Hey @fzyzcjy, first off, a **HUGE** CONGRATULATIONS! A year has flown by, any updates on the ETA now?

**Reply by fzyzcjy** [reply]
@AndryHTC Hi thank you! There has not been much progress yet.
